### ISSUE TITLE
Intermittent Internal Flavor Posts

### ISSUE THREAD

**Issue by Treasa729** [SpadesNeil/VOREstation#262]
Considering that we now have a feature where prey who've been eaten by pets and/or other mobs get little posts every minute or so, I think that a feature like that would be a fun thing to add to something players can do as well. Allow players to set up their own little flavor posts that any prey will get while in their belly/[insertorganhere] from time to time, and make it a toggle on/off feature so that if you are more involved in something without time to post, your prey can still get something to enjoy in the meantime, or when the scene is unaware or the like, but it wouldn't get in the way of when the scenes are still actively going on.

"[2:53:26 AM] Arokha: Make a git issue, include in the issue that I told you to make it and I'd do it if nobody else does
[2:53:34 AM] Arokha: hell just copypaste that quote"

**Comment by nightwingthedragon** [comment]
Well, the thing is, while that's all well and good, it does also decrease player-player interactions. At present, with the mechanics the way they are, if you want to scene with someone, you send them posts. Whereas with this, you don't have to send so many once they're inside you. It kinda... Doesn't seem like it'd be good for RP.

**Comment by Arokha** [comment]
I can think of a few reasons for this though. I know several people, a few admins included, who like being just carried around and kept as bellypet throughout a few hours. Gives them something to look at while their pred goes about their business.

When I've done this to other people, i've usually stopped in a secluded area and typed up a quick post to them, but often I'm getting dragged around to check things or heal people so...

**Comment by Treasa729** [comment]
Yeah, it's definitely not something I'd use often, but there are times when you're carrying someone in your gut while you /can't/ type up posts. It would decrease player-player interaction for some, but for others, not only would it not decrease the interactions, it would give the prey something to enjoy if the pred is too busy to post. There are many situations where this would come in handy, namely when you're not actively scening. Anything from the pred needing to go afk for a little bit and thus being able to turn these on to give the prey a little something to enjoy, or if a HoS has a full gut, and is chasing down a criminal, doctor is treating a patient. In my opinion it would be good for RP, it gives the prey something to do when they'd otherwise just be doing nothing at all. If you're hunting down a criminal, or treating a patient, stopping to type up a nice post isn't an option, and something like this gives a way for the pred to feel better about going about their actual jobs if they're needed instead of just ignoring it and continuing to scene. This would likely reduce the amount of people who neglect their actual jobs for scenes, and provide an entirely new aspect to RP. You could be going about surgery and still 'posting' back and forth with your prey as they respond to your automated posts so that the scene doesn't stop in the dorm or wherever, it continues when you're too busy to post back and stuff! When you're free once more, you can go right back to typing out your actual posts for prey. On top of this, it could do a lot to extend scenes, since, when they might normally end and just become the pred carrying the prey around for the most part, there would suddenly still be posts going back and forth, opening up more things for the prey to react to, which in turn gives the pred more to react to.

TL;DR : It would be very good for when you can't normally RP for some circumstance or other, and for keeping the RP going after a scene might normally just end.
